biomaj_download/download/direct.py

```diff
@@ -169,16 +180,18 @@ def list(self, directory=''):
 
             try:
                 self.crl.perform()
-                errcode = self.crl.getinfo(pycurl.RESPONSE_CODE)
-                if int(errcode) not in self.ERRCODE_OK:
+                errcode = int(self.crl.getinfo(pycurl.RESPONSE_CODE))
+                if errcode == 405:  # HEAD not supported
+                    msg = 'Listing ' + file_url + ' not supported. This is fine, continuing.'
+                    self.logger.info(msg)
```

Comment:
I think we should continue in loop in this case (err not supported) else will continue in code to get date/size info

Reply:
I agree with the logic but I was wondering what will happen later in the code if those information are missing.

Reply:
If head is not supported, we  won t be able to extract time etc.. info anyway from curl response so next lines of code should fail.
After that, if not set, download should always be done (for direct).
This was the behavior before. List was an attempt to get metadata to decide if data was updated or not. If we can t get info, we download file, meaning all bank updates will download the file.

Reply:
Just to be sure I tested what would happen for a bank based on such servers (i.e. I created a `directhttp` bank to `httpbin.org/post`). Update seems fine so I think it's OK to `continue` in loop (which in fact should exit the loop since there is only one file).

Reply:
Added `continue` in 068f034.